pyproject.toml

```diff
@@ -1,13 +1,16 @@
 [tool.poetry]
-name = "python3-saml"
+name = "maykin-python3-saml"
 version = "1.16.0"
 description = "Saml Python Toolkit. Add SAML support to your Python software using this library"
 license = "Apache-2.0"
-authors = ["SAML-Toolkits <contact@iamdigitalservices.com>"]
+authors = [
+    "SAML-Toolkits <contact@iamdigitalservices.com>",
+    "Maykin Media <support@maykinmedia.nl>",
+]
```

Comment:
Probably better to leave `authors` as is and move maykinmedia to `maintainers`

Reply:
it's a different package name with additions authored by us, we don't maintain the python3-saml library at all, only our own stuff